src/topology/builder.rs

```diff
@@ -623,9 +610,14 @@ impl Runner {
                             let mut t = self.transform.clone();
                             let mut outputs_buf = self.outputs.new_buf_with_capacity(events.len());
                             let task = tokio::spawn(async move {
-                                for event in events {
-                                    t.transform(event, &mut outputs_buf);
+                                t.transform_all(events, &mut outputs_buf);
+                                /*
+                                for array in events {
+                                    for event in array.into_events() {
+                                        t.transform(event, &mut outputs_buf);
+                                    }
                                 }
+                                 */
```

Comment:
There seems to be some accidental leftover.

Reply:
Fixed in a59a99cc9